storm-core/test/clj/org/apache/storm/clojure_test.clj

```diff
@@ -19,7 +19,9 @@
            [org.apache.storm.tuple Fields])
   (:use [org.apache.storm testing clojure config])
   (:use [org.apache.storm.daemon common])
-  (:require [org.apache.storm [thrift :as thrift]]))
+  (:require [org.apache.storm.internal [thrift :as thrift]])
+  (:import [org.apache.storm Thrift])
+  (:import [org.apache.storm.utils Utils]))
```

Comment:
Instead of changing this file can we move it into storm-clojure and be sure it works without modifications?